docs/1.0. Certificate Provisioning.md

```diff
@@ -221,126 +225,128 @@ If an EST Client implements the AMWA IS-04 Node API, then the EST Client SHOULD
+An EST Client SHOULD periodically check the revocation status of both the Root CA and their(its?) TLS Certificates using [OCSP][RFC-6960] and [CRL][RFC-5280]. If a TLS Certificate is revoked, the EST Client MUST stop using the revoked certificate immediately and follow [Initial Certificate Provisioning](#initial-certificate-provisioning) workflow to replace the certificate.
 
-## Notes to Implementers
+## Notes(?) to Implementers
+(MOVE TO IMPLEMENTATION GUIDE?)
 
-- EST Clients MUST have access to a good source of randomness in an acceptable amount of time, for more information on achieving this see [BRSKI - Section 5.9.4](https://tools.ietf.org/html/draft-ietf-anima-bootstrapping-keyinfra-41#section-5.9.4) and [RFC 4086 - Randomness Requirements for Security][RFC-4086].
-- The certificate returned by the EST Server MAY not be valid until sometime in the future
-  - The 'Not Before' date MUST be checked before using the certificate
-- Care SHOULD be taken when implementing the certificate renewal process, to make sure the renewal does not affect user operation in a critical way (eg. loss of control by connected NMOS clients or affect the primary operation of the device).
-- Some Base64 decoding libraries expect a line length of 64 characters, however this is not required by [RFC 4648 - Section 3.1](https://tools.ietf.org/html/rfc4648#section-3.1) and some EST Server implementations do not limit the line length of encoded data.
-  - EST Clients MUST be able to decode Base64 data that is not limited to 64 characters per line
+- EST Clients MUST have access to a good source of randomness in an acceptable amount of time. For more information on achieving this, see [BRSKI - Section 5.9.4](https://tools.ietf.org/html/draft-ietf-anima-bootstrapping-keyinfra-41#section-5.9.4) and [RFC 4086 - Randomness Requirements for Security][RFC-4086].
+- The certificate returned by the EST Server MAY not be valid until some time in the future.
+  - The 'Not Before' date MUST be checked before using the certificate.
+- Care SHOULD be taken when implementing the certificate renewal process, to make sure the renewal does not affect user operation in a critical way (e.g. loss of control by connected NMOS Clients or the primary operation of the device).
+- Some Base64 decoding libraries expect a line length of 64 characters, although this is not required by [RFC 4648 - Section 3.1](https://tools.ietf.org/html/rfc4648#section-3.1) and some EST Server implementations do not limit the line length of encoded data.
+  - EST Clients MUST be able to decode Base64 data that is not limited to 64 characters per line.
```

Comment:
Suggested moving this to the Certificate Request Response section, moving to the Implementation Guide or deleting